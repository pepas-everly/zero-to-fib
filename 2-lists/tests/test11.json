../../1-atoms/tests/test11.json