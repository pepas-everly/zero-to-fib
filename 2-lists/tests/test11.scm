../../1-atoms/tests/test11.scm