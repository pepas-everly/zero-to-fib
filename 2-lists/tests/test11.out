../../1-atoms/tests/test11.out